src/components/Login/LoginFormWalletFile/LoginFormWalletFile.scss

```diff
@@ -4,4 +4,12 @@
     padding-top: 16px;
     border-top: 1px solid rgba(#000, 0.1);
   }
+
+  .listWallet{
+    width: 100%;
+
+    padding-top: 16px;
+    // float:left;
+  }
+
 }

```

Comment:
As seen with @deanpress this will change once a class for the <select> will be available.